features/dd-sdk-android-session-replay/build.gradle.kts

```diff
@@ -103,11 +104,16 @@ dependencies {
     }
     testImplementation(libs.bundles.jUnit5)
     testImplementation(libs.bundles.testTools)
+    unmock(libs.robolectric)
 
     // TODO MTG-12 detekt(project(":tools:detekt"))
     // TODO MTG-12 detekt(libs.detektCli)
 }
 
+unMock {
+    keep("android.widget.ImageView\$ScaleType")
```

Comment:
why do we needed this ?

Reply:
otherwise running unit test fails, because probably `android.jar` for API 34 changed behaviour of `ScaleType.values()` which returns null and then `when` branch fails to process this value in the `getDrawableScaledDimensions`